berkeley-mobile/Resources/Covid/CovidResourceViewController.swift

```diff
@@ -271,3 +276,12 @@ extension CovidResourceViewController {
         presentAlertLinkUrl(title: "Are you sure you want to open Safari?", message: "Berkeley Mobile wants to navigate to the Daily Symptom Screener", options: "Cancel", "Yes", website_url: url)
     }
 }
+
+// MARK: - Analytics
+
+extension CovidResourceViewController {
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        Analytics.logEvent("opened_covid_resource", parameters: [:])
+    }
+}
```

Comment:
@sidthesquid new event for opening the Covid-19 tab

Reply:
Oki ty will follow along